CHANGELOG.rst

```diff
@@ -1,3 +1,21 @@
+********************
+[0.2.2] - 2021-0X-XX
+********************
+
+**Bugfixes**:
+
+- Placement of mutations at non-inference sites was not always truely parsimonious,
```

Comment:
truly

Reply:
Maybe "fully parsimonious" would be a better description. Also this won't make sense to an outsider: "Mutations at non-inference sites are now guaranteed to be fully parsimonious. Previous versions required a mutation above the root when the input ancestral state disagreed with the ancestral state produced by the parsimony algorithm."

Reply:
Yep, sounds good. I was probably dozy when I wrote it.